grammar.js

```diff
@@ -948,18 +954,44 @@ module.exports = grammar({
     ),
 
     insert: $ => seq(
-      choice($.keyword_insert, $.keyword_replace),
-      $.keyword_into,
+      choice(
+        $.keyword_insert,
+        $.keyword_replace
+      ),
+      optional(
+        choice(
+          $.keyword_low_priority,
+          $.keyword_delayed,
+          $.keyword_high_priority,
+        ),
+      ),
+      optional($.keyword_ignore),
+      optional($.keyword_into),
       $.table_reference,
       optional(
         seq(
           $.keyword_as,
           field('table_alias', $._alias_identifier)
-        )
+        ),
       ),
       choice(
         $._insert_values,
-        $._insert_set,
+        $._set_values,
+      ),
+      optional(
+        seq(
+          $.keyword_on_conflict,
+          optional(
```

Comment:
This optional is not necessary. Otherwise it would be possible to end the statement with `on conflict`

Reply:
Fixed